app/controllers/concerns/foreman/controller/provisioning_templates.rb

```diff
@@ -12,7 +12,21 @@ def load_vars_from_template
 
   def find_global_registration
     template_name = Setting[:default_global_registration_item]
-    @provisioning_template = ProvisioningTemplate.find_by(name: template_name)
+    @provisioning_template = ProvisioningTemplate.unscoped.find_by(name: template_name)
+    @global_registration_vars = global_registration_vars if @provisioning_template
+  end
+
+  def global_registration_vars
+    permitted = Foreman::Plugin.all
+                               .map(&:global_registration_vars)
+                               .flatten.compact.uniq
+    {
+      user: User.current,
+      auth_token: User.current.jwt_token!(expiration: (4 * 60 * 60)),
+      organization: Organization.find_by(id: params['organization_id']),
```

Comment:
We need to add authorization here, meaning we must add `.authorized` scope with the correct permission.